addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/AnalyticsSteps.java

```diff
@@ -0,0 +1,41 @@
+package io.fabric8.launcher.osio.steps;
+
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import io.fabric8.launcher.core.api.events.StatusMessageEvent;
+import io.fabric8.launcher.osio.client.AnalyticsClient;
+import io.fabric8.launcher.osio.projectiles.OsioLaunchProjectile;
+import okhttp3.FormBody;
+import okhttp3.Response;
+import org.apache.maven.model.Dependency;
+
+import static io.fabric8.launcher.core.api.events.StatusEventType.GITHUB_PUSHED;
+
+@Dependent
+public class AnalyticsSteps {
+
+    @Inject
+    private AnalyticsClient analytics;
+
+    public void pushToGithubRepository(OsioLaunchProjectile projectile) {
+        FormBody.Builder requestBody = new FormBody.Builder();
+        for (Dependency dep : projectile.projectDependencies()) {
+            if (dep.getVersion() != null) {
+                requestBody.add("dependency", dep.getGroupId() + ":" + dep.getArtifactId() + ":" + dep.getVersion());
+            } else {
+                requestBody.add("dependency", dep.getGroupId() + ":" + dep.getArtifactId());
+            }
+        }
+        requestBody.add("gitRepository", projectile.getGitRepositoryName());
+
+        if (projectile.getGitOrganization() != null) {
+            requestBody.add("gitOrganization", projectile.getGitOrganization());
+        }
+
+        Response response = analytics.analyticsRequest("/api/v1/empty-booster", requestBody.build());
```

Comment:
The `Response` object is not used, no need to expose it

Reply:
I think the  `analyticsRequest`  method doesn't need to return anything. If any error happens, it should be thrown in the method directly

Reply:
Yes, you can use that inside the `AnalyticsClient` class